assets/src/edit-story/components/fontPicker/pickerContainer.js

```diff
@@ -68,6 +69,18 @@ const Divider = styled.hr`
   border-width: 1px 0 0;
 `;
 
+const SearchInput = styled.input`
+  margin: 8px;
+  padding: 4px;
+  width: 100%;
+  border-radius: 4px;
+  border: 1px solid ${({ theme }) => theme.colors.bg.v5};
+  font-size: ${({ theme }) => theme.fonts.input.size};
+  line-height: ${({ theme }) => theme.fonts.input.lineHeight};
+  font-weight: ${({ theme }) => theme.fonts.input.weight};
+  font-family: ${({ theme }) => theme.fonts.input.family};
```

Comment:
Do we delay switching to the proper (dark-mode) design until the last font picker PR or?

Reply:
That’s up to you. I’m just following the design in the ticket.